app/testOnly/views/TestOnlyStartPage.scala.html

```diff
@@ -724,6 +724,117 @@
 }
 }
 
+@siaContent = {
+@formWithCSRF(action = testOnlyRoutes.StartJourneyController.startJourneySiaSubmit) {
+
+    @signInAsContent
+    @enrolmentsContent
+
+    @govukInput(Input(
+        id = "siaTotalAmount",
+        name = "siaDebtTotalAmount",
+        label = Label(
+            isPageHeading = false,
+            classes = "govuk-label--m",
+            content = Text("Enter total debt amount without interest")
+        ),
+        classes = "govuk-input--width-5",
+        spellcheck = Some(false),
+        prefix = Some(PrefixOrSuffix(
+            content = Text("£")
+        )),
+        errorMessage = form.error("siaDebtTotalAmount").map(e => ErrorMessage.errorMessageWithDefaultStringsTranslated(
+            content = Text(e.message)
+        )),
+        value = form.data.get("siaDebtTotalAmount").orElse(Some("10000")),
+        hint = Some(Hint(content = Text(s"Enter an amount between £1 and ${appConfig.PolicyParameters.SIA.maxAmountOfDebt.gdsFormatInPounds} for SIA.")))
+    ))
+
+    @interestAmountContent
+
+    @govukInput(Input(
+        id = "siaNINO",
```

Comment:
I'm not sure the `id` and `name` here are consistent with what you've written in the actual form logic. In the form you're looking for `siaTaxReference` which is equal to `siaTaxReference`. Perhaps it could be more generic with some like `nino` since we could in theory be looking for nino's for other tax regimes in the future